some updates (#2760)
Azure/azure-sdk-for-cpp: sdk/keyvault/azure-security-keyvault-secrets/test/samples/sample1-basic-operations/sample1-basic-operations.md

## Getting a Secret

> To get a secret from the key vault  you will need to call GetSecret.

Reviewer: ```suggestion
Call GetSecret to retrieve a secret from Key Vault.
```
